app/assets/stylesheets/style.scss

```diff
@@ -1,3 +1,31 @@
+body {
+  background-image: url("http://i.imgur.com/y6o43z3.jpg");
+  background-repeat: no-repeat;
+  background-position: center top;
+}
+
+#footer_margin {
```

Comment:
Avoid using id selectors<br>Selector `footer_margin` should be written in lowercase with hyphens